pkgs/tools/system/dell-command-configure/default.nix

```diff
@@ -55,7 +55,12 @@ let
     pname = "dell-command-configure-unpacked-wrapper-lib";
     inherit version;
 
-    src = ./.;
+    src = lib.fileset.toSource {
+      root = ./.;
+      fileset = lib.fileset.unions [
+        ./wrapper-lib.c
+      ];
+    };
```

Comment:
Why not just copy the one file?

Reply:
you mean something like?:
```
unpackPhase = ''
  cp ${./wrapper-lib.c} wrapper-lib.c
'';
```

Reply:
Seems to work. Please let me know if this was the suggested way.

Reply:
👍🏼